src/EnergyPlus/OutputProcessor.cc

```diff
@@ -1672,6 +1672,14 @@ namespace OutputProcessor {
 		cCurrentModuleObject = "Meter:Custom";
 		NumCustomMeters = GetNumObjectsFound( cCurrentModuleObject );
 
+        //make list of names for all Meter:Custom since they cannot refer to other Meter:Custom's
+		std::vector<std::string> namesOfMeterCustom;
```

Comment:
FWIW,  I just looked at an example input file produced by BEopt and we have over 30 Meter:Custom objects in it. (That said, we eventually would like to move away from using so many. For example, now that E+ supports oil/propane heating coils and equipment, we can remove a bunch.)